<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeurWidget</class>
 <widget class="QWidget" name="ServeurWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServeurWidget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelNumeroDePort">
        <property name="text">
         <string>Num√©ro de port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="SpinBoxNumeroDePort">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>5000.000000000000000</double>
        </property>
        <property name="value">
         <double>1025.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonLancerServeur">
      <property name="text">
       <string>Lancement serveur</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelClients">
        <property name="text">
         <string>Clients</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditClients"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonQuitter">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ServeurWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
